lib/operations/explain.js

```diff
@@ -0,0 +1,27 @@
+'use strict';
+
+const Aspect = require('./operation').Aspect;
+const CoreCursor = require('mongodb-core').Cursor;
+const defineAspects = require('./operation').defineAspects;
+const OperationBase = require('./operation').OperationBase;
+
+class ExplainOperation extends OperationBase {
+  constructor(cursor) {
+    super();
+
+    this.cursor = cursor;
+  }
+
+  execute() {
+    const cursor = this.cursor;
+
+    if (cursor._initImplicitSession) {
+      cursor._initImplicitSession();
+    }
+    return CoreCursor.prototype.next.apply(cursor, arguments);
```

Comment:
this looks suspiciously like not an operation 🤔

Reply:
The original `Cursor.prototype.explain` uses `executeOperation` with this code, but I do agree that this doesn't look like an operation.